integrations/gnu_libmicrohttpd/CMakeLists.txt

```diff
@@ -14,8 +15,11 @@ add_library(prometheus-http SHARED prometheus_microhttpd.cc)
 
 target_link_libraries(prometheus-http PUBLIC
   ${MHD_LIBRARIES} prometheus-client)
-set_target_properties(prometheus-http PROPERTIES VERSION "0")
-set_target_properties(prometheus-http PROPERTIES SOVERSION "0.0.0")
+set_target_properties(prometheus-http PROPERTIES
+                      VERSION "0"
+                      SOVERSION "0.0.0"
+                      CXX_STANDARD 11
```

Comment:
This is a CMake 3.x statement as far as I can tell from the manual, so you would have to bump cmake_required_version. Is it required or just nice to have?